ecs-logs-collector.sh

```diff
@@ -797,6 +806,80 @@ enable_ecs_agent_debug() {
   fi
 }
 
+disable_docker_debug() {
+  try "disable debug mode for the Docker daemon"
+
+  if [ -e /etc/sysconfig/docker ] && ! grep -q "^\\s*OPTIONS=\"-D" /etc/sysconfig/docker; then
+    info "Debug mode is already disabled."
+  else
+
+    if [ -e /etc/sysconfig/docker ]; then
+      case "${init_type}" in
+        systemd)
+          # sed -i 's/^OPTIONS="\(.*\)/OPTIONS="-D \1/g' /etc/sysconfig/docker
+          sed -i 's/\-D//g' /etc/sysconfig/docker
+          ok
```

Comment:
this seems to be common to both cases (systemd and *). can we move this to be before the case and remove it here and L829 below?